keyboards/cannonkeys/satisfaction75/rev1/rev1.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
+
+#include "satisfaction75.h"
+
+// This layout is the default - it's what's in VIA
+// ANSI, bottom row 1.25/1.25/1.25/6.25/1/1/1
+#define LAYOUT_default( \
+	K000,       K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013,             \
+	K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113,       K115, \
+	K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213,       K215, \
+	K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311,       K313,       K315, \
+	K400,       K402, K403, K404, K405, K406, K407, K408, K409, K410, K411, K412, K413,       K415, \
+	K500, K501, K502,             K505,                   K509, K510, K511, K512, K513,       K515  \
+) { \
+	{ K000,  KC_NO, K002,  K003,  K004,  K005,  K006,  K007,  K008,  K009,  K010,  K011,  K012,  K013,  KC_NO,  KC_NO }, \
+	{ K100,  K101,  K102,  K103,  K104,  K105,  K106,  K107,  K108,  K109,  K110,  K111,  K112,  K113,  KC_NO,  K115 }, \
+	{ K200,  K201,  K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213,  KC_NO,  K215 }, \
+	{ K300,  K301,  K302,  K303,  K304,  K305,  K306,  K307,  K308,  K309,  K310,  K311,  KC_NO, K313,  KC_NO,  K315 }, \
+	{ K400,  KC_NO, K402,  K403,  K404,  K405,  K406,  K407,  K408,  K409,  K410,  K411,  K412,  K413,  KC_NO,  K415 }, \
+	{ K500,  K501,  K502,  KC_NO, KC_NO, K505,  KC_NO, KC_NO, KC_NO, K509,  K510,  K511,  K512,  K513,  KC_NO,  K515 } \
+}
+
+// ISO, bottom row 1.25/1.25/1.25/6.25/1/1/1
+#define LAYOUT_ISO( \
```

Comment:
convention is to do `LAYOUT_some_lowercase_description`

```suggestion
#define LAYOUT_iso( \
```